### ISSUE TITLE
App crashes after expanding JavaScript Map object in Console

### ISSUE THREAD

**Issue by idioqa** [a85/POSTMan-Chrome-Extension#10437]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

App crashes when try to expand Map object in JavaScript Console.

### Steps To Reproduce

1. Open my collection https://github.com/idioqa/Postman/blob/99ff55de71b84e630a3a4d329cb7dfff9baeaa5e/crash.postman_collection.json#L331 or use code like this in your collection
`const reqData = new Map();
    pm.request.url.query.map((el) => {
        reqData.set({el});
    })
    console.log(reqData);`
2. Send request. (request req6_get from mine collection or any other collection with code above)
3. Open Console
4. Press "arrow down" to expand Map in console. (Marked with red rectangle)

### Screenshots or Videos

![Postman_nIvMXg8M0w](https://user-images.githubusercontent.com/93031345/141381503-eddcfeff-fb20-448a-8d7c-9ae54a33f10c.png)

https://user-images.githubusercontent.com/93031345/141383555-5bb3cb22-4848-41a1-bc03-1981c2a181fa.mp4

### Environment Information

```markdown
- Operating System: Windows 8.1
- Platform Type: Native App
- Postman Version: 9.1.3
```

**Comment by coditva** [comment]
@idioqa Thanks for writing in! I was able to replicate this issue.

While we fix the issue, as a workaround, you can stringify the `reqData` object before logging:
```javascript
console.log(JSON.stringify(reqData, null, 2));
```

We'll post updates related to the issue in this ticket itself.

**Comment by ashuvyaspm45** [comment]
@idioqa We have released a fix for this and will be available in v10.12. Please give it a try and give us feedback. Thanks for your patience.
